mne/time_frequency/tfr.py

```diff
@@ -1065,7 +1065,7 @@ def save(self, fname, overwrite=False):
 def _prepare_write_tfr(tfr, condition):
     """Aux function"""
     return (condition, dict(times=tfr.times, freqs=tfr.freqs,
-                            data=tfr.data, info=tfr.info, nave=tfr.nave,
+                            data=tfr.data, info=tfr.info._store, nave=tfr.nave,
```

Comment:
The new `Info` object doesn't inherit directly from `dict`, but from `MutableMapping`. This is the one interface breakage I couldn't fix. `isinstance(info, dict)` will return `False`.

Of course, `Info` objects have long not been `dict`'s anymore. They behave like one, but using an actual `dict` instead of an instance of `Info` will break mne-python in many places, since none of the utility functions (e.g. `_check_consistency`) will be available.

When reading and writing instances of `Info` using H5IO, we must cast the `Info` object to and from a `dict` along the way. In this line, I accomplish casting an `Info` to a `dict` object by using `info._store`: the `dict` object backing the `Info` object.

Maybe it's cleaner to use `dict(info)` as a cast? That works by the way...